types/google__maps/index.d.ts

```diff
@@ -255,7 +264,12 @@ export type Language = (
 );
 
 export type GoogleMapsClientEndpoint<Request, Response> = (query: Request, callback?: ResponseCallback<Response>) => RequestHandle<Response>;
+export type GoogleMapsClientEndpointWithPromise<Request, Response> = (query: Request, callback?: ResponseCallback<Response>) => RequestHandleWithPromise<Response>;
 
+/**
+ * Google Maps clieant that provides access to all the APIs.
+ * @param Promisable Determine if `asPromise()` is available in request. Default is WITHOUT_PROMISE.
```

Comment:
`GoogleMapsClient` is not a parametric type, this must be a leftover of the previous commit